gcloud-java-compute/src/test/java/com/google/gcloud/compute/ComputeImplTest.java

```diff
@@ -216,11 +249,33 @@
   private static final Compute.LicenseOption LICENSE_OPTION_FIELDS =
       Compute.LicenseOption.fields(Compute.LicenseField.CHARGES_USE_FEE);
 
+  // Operation options
+  private static final Compute.OperationOption OPERATION_OPTION_FIELDS =
```

Comment:
Nit: maybe it won't make a difference on this line, but in general if we import Compute.*Option and Operation., we may be able to avoid some line wrapping in this file.